.github/workflows/broken-link-check-full.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         if: steps.lychee.outputs.exit_code == 0
         run: |
             echo "🎉 No broken links found! 🎉" >> $GITHUB_STEP_SUMMARY
-            - name: Create GitHub Issue From lychee output file
```

Comment:
This was the line that I believe was causing the strange exit error as it's not a proper shell command. I think I left it in when I refactored this workflow in another PR. 🤦🏻